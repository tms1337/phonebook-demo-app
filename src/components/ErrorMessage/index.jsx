import React from "react";
import { colors } from "../../ui";

const ErrorMessage = () => {
  return <div style={styles.root}>An error has occcured</div>;
};

const styles = {
  root: {
    color: colors.DANGER
  }
};

export default ErrorMessage;
