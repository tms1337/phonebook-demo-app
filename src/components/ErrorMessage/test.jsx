import React from "react";
import renderer from "react-test-renderer";

import ErrorMessage from ".";

describe("ErrorMessage", () => {
  it("Snapshot renders", () => {
    const component = renderer.create(<ErrorMessage />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
