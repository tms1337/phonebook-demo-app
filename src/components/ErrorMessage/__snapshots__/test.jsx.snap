// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage Snapshot renders 1`] = `
<div
  style={
    Object {
      "color": "#ff1f40",
    }
  }
>
  An error has occcured
</div>
`;
